ChatSX/components/CommonAllMembers/index.tsx

```diff
@@ -430,10 +430,12 @@ const CommonAllMembers = ({navigation, chatroomID, isDM, showList}: any) => {
               body: {isToast: true, msg: `${apiResponse?.errorMessage}`},
             });
           } else {
+            const chatroomWithUser = response?.chatroom?.chatroomWithUser;
             let createdChatroomID = response?.chatroom?.id;
             if (!!createdChatroomID) {
               navigation.navigate(CHATROOM, {
                 chatroomID: createdChatroomID,
+                chatroomWithUserParam: chatroomWithUser,
```

Comment:
This will be required as currently inside the conversationSync API we are not getting the userMeta for the chatroomWithUserId present over there, so taking it from the response of dm/create API so further API calls like community/dm/status and other Chatroom details like chatroom title won't fail.

**Curl for that conversation sync API:** curl -H "accept:application/json, text/plain, */*" -H "x-sdk-source:chat" -H "x-version-code:15" -H "x-platform-code:rn" -H "authorization:Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhY2Nlc3NfdXVpZCI6IjRiZDA3MzA4LTM2ZjEtNDZjZi1iYmQxLTE3ODhlMDJiMWVjNyIsImFwaV9rZXkiOiJhODBkZjY3OS00ZmRjLTRhNGYtYjY0Ni1kYWZkMTA2MDNiNjIiLCJleHAiOjE2OTYzNTc4ODEsInVzZXJfdW5pcXVlX2lkIjoiOGE3NGQ1ZGItY2I3Ny00ZjNiLWE1NTUtOWEzOTY3MDlkNTE0In0.QDuyfOMoCkBc5clpnUqmFpzW8E1fWylBzymw4DuTHsY" https://betaauth.likeminds.community/conversation/sync?page=1&page_size=500&chatroom_id=87924&max_timestamp=1696354296371&min_timestamp=0